### PR TITLE
PO Review before 7.11 Release

### PR DESCRIPTION
**Pull request by florentsarat**
Getting ready for the 7.11

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -35,27 +53,50 @@ Extract informations from a project sources to generate an asciidoc document. Th
...
+Bonita offers maven archetypes to help bootstrap these extension points
 
+More information:
+* [Bonita actor filter archetype](actor-filter-archetype.md)
+* [Bonita connector archetype](connector-archetype.md)
 
-### Runtime
+::: info
+This is not limited to 7.11 and can be used from version 7.7
+:::
```
Comment (NathalieC) [inline md/release-notes.md:72]: Has this been added to documentation of versions 7.7 to 7.10?

File: md/release-notes.md [hunk 2]
```diff
@@ -35,27 +53,50 @@ Extract informations from a project sources to generate an asciidoc document. Th
...
+* Potential: total number of potential couples, prior to duplicates removal. If this counter grows faster than the executed one, this indicates that we have lot of duplicates. See bonita.log for ids of duplicate couples.
+* ReTriggerMatchingTasks: number of times we detect that there are more potential matches to be processed.
+We retrieve the matches by batch of 100, a retrigger is performed if there are remaining potential matches. This counter increases when they are remaining messages to be processed. If this counter continues to increase, this means that we are continuously processing matches (even if no new waiting events and/or messages have been created), so events processing is delayed ie new events/messages are not processed on the fly as it should in a nominal situation.
+
+##### Connectors
+There is now a configurable time threshold on connectors. When connector execution takes longer than the threshold, a warning is logged. It allows identification of slow running services 
+The parameter is :
+```
+bonita.tenant.connector.warnWhenLongerThanMillis
+```
```
Comment (NathalieC) [inline md/release-notes.md:95]: Can you link to the configuration page where how to configure connectors is explained?

File: md/release-notes.md [hunk 3]
```diff
@@ -35,27 +53,50 @@ Extract informations from a project sources to generate an asciidoc document. Th
...
+
+#### Bonita Purge Tool
```
Comment (NathalieC) [inline md/release-notes.md]: I'd see that as a new value more than an improvement.

File: md/release-notes.md [hunk 4]
```diff
@@ -35,27 +53,50 @@ Extract informations from a project sources to generate an asciidoc document. Th
...
+A new open source project is available [here](https://github.com/bonitasoft/bonita-purge-tool)
+It cleans archived process instances older than a specific date. Allows to (potentially) reduce the size of a production database.
 
 ## Bundle changes
 
-Upgrade Tomcat from 8.5.47 to 8.5.53 (tomcat-dbcp from 9.0.16 to 9.0.31) **subject to change prior GA**
+Upgrade Tomcat from 8.5.47 to 8.5.53 (tomcat-dbcp from 9.0.16 to 9.0.31)
```
Comment (NathalieC) [inline md/release-notes.md:101]: For me, it should go to "Technical updates".

File: md/release-notes.md (hunk 1 above)
Comment (florentsarat) [reply md/release-notes.md:72]: Yes

File: md/release-notes.md (hunk 4 above)
Comment (pabloalonso) [reply md/release-notes.md:101]: It is Technical updates of bundle not Bonita, that is why is in this section. Added a new section Technical updates to Bundle section

File: md/release-notes.md (hunk 2 above)
Comment (pabloalonso) [reply md/release-notes.md:95]: Done

File: md/release-notes.md (hunk 3 above)
Comment (pabloalonso) [reply md/release-notes.md]: done

**Pull request merged**